lib/uo_cli.rb

```diff
@@ -545,7 +548,7 @@ def self.create_scenario_csv_file(feature_id)
           end
           # Rescue if json isn't a geojson feature_file
         rescue NoMethodError
-          abort("\nOops! You didn't provde a valid feature_file. Please provide path to the geojson feature_file")
+          abort("\nOops! You didn't provide a valid feature_file. Please provide path to the geojson feature_file")
```

Comment:
I really like my plugin/pre-commit hook that finds typos and recommends fixes ❤️